fix: react-native 0.33 compatibility

kennethlynne opened the pull request.

nadeesha committed:
fix: react-native 0.33 compatibility (#1)

nadeesha committed:
Update README.md

techrah commented on android/build.gradle:
```diff
@@ -16,5 +16,5 @@ android {
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.11.+'
+    compile 'com.facebook.react:react-native:0.33.+'
```
Does this work? The latest version should be 0.20.1 according to https://mvnrepository.com/artifact/com.facebook.react/react-native. PR #3 seems to be the correct fix. Note also that these changes have been required since RN 0.29.0.

sondremare committed:
fix: support React Native versions over 0.33

sondremare committed:
fix: make compatible with RN 0.47

BREAKING CHANGE: will not work for RN 0.46 and lower

sondremare committed:
chore: release version 1.0.0

sondremare committed:
fix: support react native 0.57

React native 0.57 uses gradle 3, which in turn requires higher build tools and sdk version of android. So we bump it here in order to prevent build failures.

sondremare committed:
chore: release version 2.0.0
